rsocket-broker-client-spring/src/main/java/io/rsocket/broker/client/spring/BrokerClientAutoConfiguration.java

```diff
@@ -73,7 +69,7 @@ public BrokerClientProperties brokerClientProperties() {
 	@ConditionalOnMissingBean
 	public BrokerRSocketRequesterBuilder brokerRSocketRequesterBuilder(
 			RSocketConnectorConfigurer configurer, RSocketStrategies strategies,
-			BrokerClientProperties properties) {
+			BrokerClientProperties properties, @Autowired(required = false) BrokerClientMimeTypeSupplier brokerClientMimeTypeSupplier) {
```

Comment:
I wonder if it should be a `MimeType` field in `BrokerClientProperties`?

Reply:
Could be as well. If you think that's better, I'll switch it.